### PR TITLE
[4.0] Array to php 5.4+ notation [Plugins 5: System]

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Improvement.

### Summary of Changes

Since 4.0 doesn't support php 5.3 anymore, use php 5.4+ array notation in System plugins.

### Testing Instructions

Simple code review.

### Documentation Changes Required

None.

### CODE REVIEW

File: plugins/system/debug/debug.php [hunk 1]
```diff
@@ -597,13 +597,13 @@ protected function displayErrors()
...
-		$html = array();
+		$html = [];
 
-		$htmlMarks = array();
+		$htmlMarks = [];
 
 		$totalTime = 0;
 		$totalMem = 0;
-		$marks = array();
+		$marks = [];
```
Comment (Quy) [inline plugins/system/debug/debug.php]: Align

File: plugins/system/debug/debug.php [hunk 2]
```diff
@@ -822,18 +822,18 @@ protected function displayQueries()
 			$totalBargraphTime = 1;
 		}
 
-		$bars = array();
-		$info = array();
+		$bars = [];
+		$info = [];
 		$totalQueryTime = 0;
-		$duplicates = array();
+		$duplicates = [];
```
Comment (Quy) [inline plugins/system/debug/debug.php]: Align

File: plugins/system/debug/debug.php (hunk 1 above)
Comment (andrepereiradasilva) [inline plugins/system/debug/debug.php]: all aligned

**Comment by Bakual** [comment]
Asking stupid, is there any reason for this change beside that it is possible now?
Eg is it faster this way?

**Comment by laoneo** [comment]
The same question popped up in my mind.

**Comment by rdeutz** [comment]
I think it is more a question of code style beside the fact we save 5 bytes :-)

**Comment by Bakual** [comment]
So do we have a new codestyle rule for using short array syntax?
Otherwise I would say to not merge those PRs because they just will create conflicts and not improve anything.

**Comment by rdeutz** [comment]
> So do we have a new codestyle rule for using short array syntax?

Not that I know.

**Comment by laoneo** [comment]
Or merge them, after the 4.0-dev branch got merged into staging.

**Comment by rdeutz** [comment]
@andrepereiradasilva could you open an issue here: https://github.com/joomla/coding-standards so we can discuss the change and then move forward whatever the decision will be

**Comment by andrepereiradasilva** [comment]
The change is code style only. OK @rdeutz

**Comment by andrepereiradasilva** [comment]
Done https://github.com/joomla/coding-standards/issues/192

**Pull request closed without merging**